python-regression/RunLocalTests.sh

```diff
@@ -0,0 +1,85 @@
+#!/bin/bash
+
+printf "Begin local testing\n"
```

Comment:
I think it's better to separate this tool into a new PR.
it feels to me like it still needs some tweaking - it misled me a few times :)